### PR TITLE
EDU-576 Make registration procedure clearer

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-576

Unifies english terms for "log in/on" and "log off/out" to simply "login" and "logout"

### CODE REVIEW

File: src/components/pages/register.yml [hunk 1]
```diff
@@ -7,9 +7,16 @@ termsAndConditionsConfirmation:
...
   de: Registrieren
-registrationInProgress:
-  en: We have sent you an email. There you will find the next steps in order to complete the registration.
-  de: Wir haben Ihnen eine E-Mail geschickt. Dort finden Sie die nächsten Schritte, um Ihre Registrierung abzuschließen.
-redirectMessage:
-  en: You will be redirected to the <0>logon page</0> in {seconds, plural, =1 {# second} other {# seconds}}.
-  de: Sie werden in {seconds, plural, =1 {# Sekunde} other {# Sekunden}} auf die <0>Anmeldeseite</0> weitergeleitet.
+registrationConfirmationHeader:
+  en: One more step remaining…
+  de: Ein Schritt fehlt noch…
```
Comment (adrianaluchian) [inline src/components/pages/register.yml:12]: A space between "remaining" / "noch" and "..." would make it a bit more readable I feel.

**Comment by ahelmberger** [comment]
> Definitely looks more readable than what we had before +1
> 
> I could even suggest adding even a bit more visibility to the message with:
> 
> * a bigger space between logo and message (I tried 100px)
> * writing the sentences on separate lines
> * emphasizing the 24h part
> 
> Something like this: Screen Shot 2022-05-24 at 16 21 21
> 
> What say you?

I say "hell yeah" (which means: done!)

File: src/components/pages/complete-password-reset.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ passwordChangedSuccessfully:
   en: Your password was changed successfully.
   de: Ihr Kennwort wurde erfolgreich geändert.
 redirectMessage:
-  en: You will be redirected to the <0>logon page</0> in {seconds, plural, =1 {# second} other {# seconds}}.
+  en: You will be redirected to the <0>login page</0> in {seconds, plural, =1 {# second} other {# seconds}}.
```
[the remark this answers is not in the document]
Comment (ahelmberger) [reply src/components/pages/complete-password-reset.yml:8]: Me too

File: src/components/pages/register.yml (hunk 1 above)
Comment (ahelmberger) [reply src/components/pages/register.yml]: Done, looks indeed better, thanks!

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**